src/components/WhoWeAre/Team.tsx

```diff
@@ -1,41 +1,165 @@
 import React from 'react';
 import styled from 'styled-components';
 import Image from 'react-bootstrap/Image';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Accordion from 'react-bootstrap/Accordion';
+import LinkedinLogo from '../../images/icons-footer/social_linkedin.png';
+import Nav from 'react-bootstrap/Nav';
+import { media } from 'styled-bootstrap-grid';
+import { colors } from '../../theme/styleVars';
 
 const Team = (props: any) => {
-  const { image, key, name, pronoun, subtitle } = props;
+  const { bio, id, image, linkedin, name, pronouns, role } = props;
 
   return (
-    <MarginTeam key={key}>
-      <ShadowTeam
-        className="shadow-team"
-        roundedCircle
-        src={image}
-        width="100%"
-      />
-      <div className="text-center">
-        <TeamName>{name}</TeamName>
-        <h5>({pronoun})</h5>
-        <p>{subtitle}</p>
-      </div>
+    <MarginTeam key={id}>
+      <Accordion defaultActiveKey="0">
+        <PersonCard>
+          <TopRow className="align-items-center justify-content-center">
+            <Col md={5} xs={5}>
+              <PersonImg fluid src={image} />
+            </Col>
+            <Col md={7} style={{ padding: 0 }} xs={7}>
+              <Card.Body style={{ padding: 0 }}>
+                <PersonName>{name}</PersonName>
+                <PersonPronouns>{pronouns}</PersonPronouns>
+                <PersonRole>{role}</PersonRole>
+                {linkedin != null && (
+                  <Nav.Link
+                    href={`https://www.linkedin.com/in/${linkedin}`}
+                    style={{ padding: 0 }}
+                    target="_blank"
+                  >
+                    <BioLogo src={LinkedinLogo} />
+                  </Nav.Link>
+                )}
+                {bio != null && (
+                  <Accordion.Toggle className="button" eventKey={id}>
+                    View Bio
+                  </Accordion.Toggle>
+                )}
+              </Card.Body>
+            </Col>
+          </TopRow>
+          <Accordion.Collapse eventKey={id}>
+            <BottomRow>
+              <BioCol>
+                <BioText className="caption">
+                  {name} ({pronouns}) - {bio}
+                </BioText>
+              </BioCol>
+            </BottomRow>
+          </Accordion.Collapse>
+        </PersonCard>
+      </Accordion>
     </MarginTeam>
   );
 };
 
-const ShadowTeam = styled(Image)`
-  box-shadow: 2px 2px 10px rgba(0, 0, 29, 0.1);
-  margin-bottom: 10px;
-  margin-top: 10px;
+const PersonCard = styled(Card)`
+  border-radius: 5px;
+  width: 22rem;
```

Comment:
[nit] Again, you'll want to alphabetize the CSS properties for organization in this file